App.jsx

```diff
@@ -27,56 +23,38 @@ const App = React.createClass({
     this.setState({name: event.target.value});
   },
 
-  handleMessageChange(event) {
-    this.setState({newMessage: event.target.value});
-  },
-
   handleKeyPress(event) {
-    const {name, newMessage} = this.state;
+    const {name} = this.state;
     // If name or newMessage are blank, do not send new message
-    if (!name || !newMessage) {
+    if (!name) {
       return;
     }
 
     // If user hits Enter key, then send message to Firebase database
     // and clear out the message box
-    if (event.key === 'Enter') {
-      Firebase.sendMessage({ name: name, message: newMessage });
-      this.setState({newMessage: ''});
```

Comment:
can you keep this line as well but not have `newMessage: ''` included? If it can't compile we can just comment it out like you did for the Firebase.sendMessage